src/vcf/meta_entry.cpp

```diff
@@ -214,6 +214,13 @@ namespace ebi
             value["Type"] != "String") {
             throw new MetaSectionError{entry.line, "INFO metadata Type is not a Integer, Float, Flag, Character or String"};
         }
+
+        // Check for predefined tag AA
+        if (value["ID"] == "AA") {
+            if (value["Type"] != "String") {
```

Comment:
I like the `map` idea. how should it be declared and defined? I always forget the recommended practice for namespace constants. const static? and definition with curly braces?